auth_server/chart/docker-auth/templates/configmap.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "docker-auth.name" . }}
+data:
+  auth_config.yml: |
+    server:
+      addr: ":{{ .Values.service.targetPort }}"
+    token:
+      issuer: "{{ .Values.configmap.data.token.issuer }}"  # Must match issuer in the Registry config.
+      expiration: {{ .Values.configmap.data.token.expiration }}
+      certificate: "/config/certs/server.pem"
+      key: "/config/certs/server.key"
+    users:
```

Comment:
Doing users this way means that the chart only supports two uses out of the box. The config files are already yaml. So I would recommend that you just use `{{ .Values.config.users | toYaml | nindent 6 }}` to inject the entire users section as embedded yaml.

Reply:
Good advice, Done for that.